src/einsteinpy/symbolic/ricci.py

```diff
@@ -170,28 +168,17 @@ def __init__(self, arr, syms, config="l", parent_metric=None):
 
         """
         super(RicciScalar, self).__init__(
-            arr=arr, syms=syms, config=config, parent_metric=parent_metric
+            arr=arr, syms=syms, config=" ", parent_metric=parent_metric
```

Comment:
It was supposed to be a blank string `config=""`

Reply:
So silly mistake xD I was looking for what could give so many mistakes in the test

Reply:
Well, it was practical lesson why testing is important 😄

Reply:
resolve